index.html

```diff
@@ -151,5 +151,293 @@ <h2 class="headline-job">I’m a software developer</h2>
+          <div class="project-card--content">
+            <p class="project-title">Multi-Post Stories Gain+Glory</p>
+            <ul class="project-tools">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>Javascript</li>
+              <li>html</li>
+            </ul>
+            <button type="button" class="view-project">See Project</button>
+          </div>
+        </div>
+
+        <div class="project-card">
+          <div class="project-card--thumb"></div>
+          <div class="project-card--content">
+            <p class="project-title">Multi-Post Stories Gain+Glory</p>
+            <ul class="project-tools">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>Javascript</li>
+              <li>html</li>
+            </ul>
+            <button type="button" class="view-project">See Project</button>
+          </div>
+        </div>
+
+        <div class="project-card">
+          <div class="project-card--thumb"></div>
+          <div class="project-card--content">
+            <p class="project-title">Multi-Post Stories Gain+Glory</p>
+            <ul class="project-tools">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>Javascript</li>
+              <li>html</li>
+            </ul>
+            <button type="button" class="view-project">See Project</button>
+          </div>
+        </div>
+
+        <div class="project-card">
+          <div class="project-card--thumb"></div>
+          <div class="project-card--content">
+            <p class="project-title">Multi-Post Stories Gain+Glory</p>
+            <ul class="project-tools">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>Javascript</li>
+              <li>html</li>
+            </ul>
+            <button type="button" class="view-project">See Project</button>
+          </div>
+        </div>
+
+        <div class="project-card">
+          <div class="project-card--thumb"></div>
+          <div class="project-card--content">
+            <p class="project-title">Multi-Post Stories Gain+Glory</p>
+            <ul class="project-tools">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>Javascript</li>
+              <li>html</li>
+            </ul>
+            <button type="button" class="view-project">See Project</button>
+          </div>
+        </div>
+
+        <div class="project-card">
+          <div class="project-card--thumb"></div>
+          <div class="project-card--content">
+            <p class="project-title">Multi-Post Stories Gain+Glory</p>
+            <ul class="project-tools">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>Javascript</li>
+              <li>html</li>
+            </ul>
+            <button type="button" class="view-project">See Project</button>
+          </div>
+        </div>
+      </div>
```

Comment:
Kindly wrap your projects in a `<section>` tag. For semantic purposes and to make your page more SEO friendly.

Reply:
Thank you for reviewing my project, I will make the necessary corrections as suggested.